### ISSUE TITLE
Detection Area

### ISSUE THREAD

**Issue by mkampstra** [ccrisan/motionpie#315]
Hi ccrisan,

Is it possible to choose a certain area of detection? So you can set where the detection is set. Now detection is on whole cam view. But it records waving trees. Detected at 2.8% (22000 @1024*768)
When a person walks by it's also around 2.6%. So lowering gives more false warnings, and setting higher will miss the point of interest.

If a certain area can be set the false warnings could be reduced.

**Comment by ccrisan** [comment]
Not yet, but this is a high priority feature.

**Comment by willbird** [comment]
Yes that looks like a nice feature :-), in view of one camera I have setup is a bush that blows in the wind :-).
